components/scream/src/physics/p3/tests/p3_ice_nucleation_unit_tests.cpp

```diff
@@ -26,29 +26,30 @@ struct UnitWrap::UnitTest<D>::TestIceNucleation {
     using KTH = KokkosTypes<HostDevice>;
 
     constexpr bool   do_predict_nc = false;
+    constexpr bool   do_prescribed_CCN = false;
     constexpr Scalar inv_dt           = 9.558E-04;
```

Comment:
As I was saying before, we should probably test both do_predict_nc is false and is true.